docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -1,58 +1,57 @@
 ---
-description: How to choose and connect to a testnet
+description: How to connect to a testnet
 ---
 
 # Connect to a testnet
 
-**Prerequisites**:
-
-* Install the latest version of Teku using a [binary distribution](Install-Binaries.md)
-    or [from source](Build-From-Source.md).
-* If running validators, [install Hyperledger Besu] to connect to an Ethereum 1.0 network.
-
-!!! note
-    Any Ethereum 1.0 client can be installed to connect to the Ethereum 1.0 network, but this
-    example uses Besu.
-
-Teku allows you run a [beacon chain client only], or you can [run the beacon chain client
-with validators] on a public testnet.
+The following instructions describe the process to connect Teku to an Ethereum 2.0 testnet.
 
 !!! important
 
     This example connects to the [Witti testnet](https://github.com/goerli/Witti). Networks can
     experience stability issues and are prone to regular resets. We recommend you regularly
     check network and client documentation for updates.
 
-## Select a network
+**Prerequisites**:
```

Comment:
Nit: I don't think we include the colon on other Prerequisites heading. ie, I think it can be removed

Reply:
Done